tests/introspection/ApiSelectorTest.cs

```diff
@@ -884,11 +884,16 @@ protected virtual bool SkipInit (string selector, MethodBase m)
 			case "initWithSSID:passphrase:isWEP:":
 			case "initWithSSIDPrefix:":
 			case "initWithSSIDPrefix:passphrase:isWEP:":
+			// MapKit
+			case "initWithMaxCenterCoordinateDistance:":
+			case "initWithMinCenterCoordinateDistance:":
+			case "initExcludingCategories:":
+			case "initIncludingCategories:":
 				var mi = m as MethodInfo;
 				return mi != null && !mi.IsPublic && mi.ReturnType.Name == "IntPtr";
-			default:
```

Comment:
Is there a trick for dealing with tabs/spaces in VSMac? I keep running into these types of diffs...

Reply:
if you set the preferences to use the Mono coding style then you should not have such issue (unless your version of VSfM has bugs).